client/me/purchases/billing-history/vat-vendor-details.tsx

```diff
@@ -11,21 +11,25 @@ export function VatVendorDetails( { transaction }: { transaction: BillingTransac
 	return (
 		<li>
 			<strong>
-				{ translate( 'Vendor %(taxName)s Details', {
-					args: { taxName: vendorInfo.tax_name },
-					comment: 'taxName is a localized tax, like VAT or GST',
+				{ translate( 'Vendor %(combinedTaxName)s Details', {
```

Comment:
By changing this translated string, it will need to be retranslated, even though you haven't actually changed anything. In cases like this it's often better to leave the string unchanged even if its variable names are not totally accurate. You can just set `taxName` to the new string you've created instead of renaming it.

Reply:
I wavered on whether to change the string variable name because for most cases, the tax name isn't actually a combination of names.

So you're point just helped make the decision to change it back. Thanks for the lesson on translated strings too!